File: ios/LivenessDetection/FaceMetrics.swift
import Foundation
import CoreGraphics

struct FaceMetrics {
  let yaw: Float
  let pitch: Float
  let roll: Float
  let leftEar: Float
  let rightEar: Float
  let mouthMar: Float
  let boundingBox: CGRect?
  let timestampMs: Int64

  var avgEar: Float {
    return (leftEar + rightEar) / 2.0
  }
}

File: android/liveness/src/main/java/com/liveness/detection/FaceMetrics.kt
package com.liveness.detection

import android.graphics.RectF

data class FaceMetrics(
  val yaw: Float,
  val pitch: Float,
  val roll: Float,
  val leftEyeEar: Float,
  val rightEyeEar: Float,
  val mouthMar: Float,
  val boundingBox: RectF?,
  val timestampMs: Long,
  val imageWidth: Int,
  val imageHeight: Int,
) {
  val avgEar: Float
    get() = (leftEyeEar + rightEyeEar) / 2f
}
